core/types/bid.go

```diff
@@ -0,0 +1,107 @@
+package types
+
+import (
+	"fmt"
+	"math/big"
+	"sync/atomic"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+// BidArgs represents the arguments to submit a bid.
+type BidArgs struct {
+	// bid
```

Comment:
`// bid `==> `RawBid that from builder directly`

Reply:
fixed